pkg/drivers/unity_test.go

```diff
@@ -169,19 +178,3 @@ func csmForUnitySkipCertISFalse() csmv1.ContainerStorageModule {
 
 	return res
 }
-
-// makes a csm object with bad allowed network input
```

Comment:
why are we deleting this test?

Reply:
It looks like that function is no longer used anywhere in the code, so it was removed[ as a part of this commit](https://github.com/dell/csm-operator/commit/76cd58055b6311197af07d7d8514f4028c16160a#diff-ca65267304d0addc0427c4576aaa1411d364782a95f394f11803e3bf1b993cdf). 

I searched through the repository and `csmForUnityAllowedNetwork` does not have any other occurrences.